### ISSUE TITLE
Tests not cleaning up apiKeys

### ISSUE THREAD

**Issue by jcass77** [stormpath/stormpath-flask#83]
It looks like the Stormpath service automatically creates new ``apiKeys`` whenever an application is created.

At the moment, separate accounts are created for [each test](https://github.com/stormpath/stormpath-flask/blob/41401abe088183bb322996c6d8044092def67b1e/tests/helpers.py#L32) in ``setUp`` to allow stormpath-flask unit tests to be run in parallel.

Unfortunately, these keys are never cleaned up again and remain even after [the application and its directories are deleted](https://github.com/stormpath/stormpath-flask/blob/41401abe088183bb322996c6d8044092def67b1e/tests/helpers.py#L39-L43).

The end result is that hundreds of ``apiKeys`` are created, which become impossible to administrate using the Stormpath website.

We probably need to clean these up properly, and delete any ``apiKeys`` that are created using the ``tearDown`` method.

**Comment by rdegges** [comment]
Thanks, I'll work on this.

**Comment by jcass77** [comment]
Doesn't look like GitHub closed this issue automatically when we merged #93 for some reason?

Closing manually...

**Comment by rdegges** [comment]
It usually only does it when I merge into master. I was waiting on the other updates to that one PR before merge / release =D
